lib/cc/yaml/nodes/glob.rb

```diff
@@ -2,9 +2,6 @@ module CC
   module Yaml
     module Nodes
       class Glob < Scalar
```

Comment:
I think you need `GlobList` to map `ratings.paths` to a `Sequence` specifically of `Glob`s... it holds the `type` declaration.